backend/Services/SightingService.cs

```diff
@@ -30,5 +31,23 @@ public IEnumerable<Sighting> GetApprovedSightings()
         {
             return _sightings.GetApprovedSightings();
         }
+
+        public Sighting CreateSighting(CreateSightingRequest request)
+        {
+            //TODO
+            //logic, change request to include locationID
```

Comment:
Standard practice when leaving a TODO comment is to write up a ticket in the backlog, and include the number of the ticket in the TODO, for example:
```suggestion
            // TODO: WS-15
            //   Logic, change request to include locationID
```

Reply:
Obviously I picked a random ticket number, yours will be different 😉